### PR TITLE
Fix failing tests

### PR DESCRIPTION
**Pull request by nkryuchkov**

### CODE REVIEW

**Review by aaron-blox — APPROVED** [review]

File: qbft/spectest/tests/controller/futuremsg/cleanup.go [hunk 1]
```diff
@@ -48,6 +48,6 @@ func Cleanup() *ControllerSyncSpecTest {
 			}),
 		},
 		SyncDecidedCalledCnt: 1,
-		ControllerPostRoot:   "2a506a58b7abbbcffd6d4ee92f154c826947ce56672564155772435456315c6d",
+		ControllerPostRoot:   "cd0e7827cc4f55c6972925aa38b79050ae7f99d6083032127b9ee1fbd28caae0",
```
Comment (alonmuroch) [inline qbft/spectest/tests/controller/futuremsg/cleanup.go:51]: fix todo

Comment (alonmuroch) [reply qbft/spectest/tests/controller/futuremsg/cleanup.go:51]: also change signed messages to new format (the new func we created)

File: qbft/spectest/tests/proposal/current_round_prev_prepared.go [hunk 2]
```diff
@@ -35,7 +35,7 @@ func CurrentRoundPrevPrepared() *tests.MsgProcessingSpecTest {
 	return &tests.MsgProcessingSpecTest{
 		Name:          "proposal happy flow round > 1 (prev prepared)",
 		Pre:           pre,
```
Comment (alonmuroch) [inline qbft/spectest/tests/proposal/current_round_prev_prepared.go:37]: the round change justifications have dataRound == NoRound. 
This basically means that it's not a previously prepared proposal with previously prepared round changes.
If you do step by step you'll see it goes into proposal.go#L153

File: qbft/spectest/tests/proposal/prepared_previously_justification.go [hunk 3]
```diff
@@ -38,7 +41,7 @@ func PreparedPreviouslyJustification() *tests.MsgProcessingSpecTest {
 	return &tests.MsgProcessingSpecTest{
 		Name:          "previously prepared proposal",
 		Pre:           pre,
```
Comment (alonmuroch) [inline qbft/spectest/tests/proposal/prepared_previously_justification.go:43]: is this a duplicate of current_round_prev_prepared?

File: qbft/spectest/tests/proposal/invalid_full_data.go [hunk 4]
```diff
@@ -1,8 +1,24 @@
...
 
-import "github.com/bloxapp/ssv-spec/qbft/spectest/tests"
+import (
+	"github.com/bloxapp/ssv-spec/qbft"
+	"github.com/bloxapp/ssv-spec/qbft/spectest/tests"
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
+)
 
 // InvalidFullData tests signed commit with an invalid full data field (H(full data) != root)
```
Comment (alonmuroch) [inline qbft/spectest/tests/proposal/invalid_full_data.go:10]: change to signed proposal not commit

**Review by alonmuroch — CHANGES_REQUESTED** [review]

File: qbft/spectest/tests/proposal/prepared_previously_justification.go (hunk 3 above)
Comment (nkryuchkov) [reply qbft/spectest/tests/proposal/prepared_previously_justification.go:43]: I think it is, so I'm removing `CurrentRoundPrevPrepared` (duplicate of `PreparedPreviouslyJustification`) and `CurrentRoundPrevNotPrepared` (duplicate of `NotPreparedPreviouslyJustification`)

File: qbft/spectest/tests/proposal/current_round_prev_prepared.go (hunk 2 above)
Comment (nkryuchkov) [reply qbft/spectest/tests/proposal/current_round_prev_prepared.go]: https://github.com/bloxapp/ssv-spec/pull/157#discussion_r1114120183

**Review by alonmuroch — APPROVED** [review]

**Pull request merged**